### DISCUSSION TITLE
bookmarklets are blocked on chrome.google.com

### DISCUSSION THREAD

**Discussion by 1016915th** [3kh0/ext-remover#261] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
any way to bypass this?

**Comment by ILikeBypassing** [comment, marked as answer]
the tab can't block bookmarklets only your administator can. Check you chrome://policy to see blocked urls and see if "javascript:" is a block url. if that is blocked then you can just use console from inspect element to inject the code manually

**Reply by 1016915th** [reply]
it is a blocked url, and i can't use inspect element

**Reply by ILikeBypassing** [reply]
can you show me a video of this happening or maybe tell me more in depth
